### ISSUE TITLE
File encryption independent of screenlock.

### ISSUE THREAD

**Issue by abhishekabhi789** [TeamAmaze/AmazeFileManager#1701]
Presently a warning shows whenever i try to encrypt some files. I encrypt and upload files to Drive. Because of this warning I've use ES file explorer to encrypt the file because i can decrypt them later after changing lock screen password. So please try to change encryption method that work independent of lockscreen password.

**Comment by EmmanuelMess** [comment]
I was thinking of encryption with a compression library, setting compression ratio to 0.

**Comment by VishalNehra** [comment]
I don't think this is possible in android environment. Sorry for responding this late as I didn't notice this issue.

**Comment by EmmanuelMess** [comment]
This is a very big problem, I don't think it should go unfixed.

**Comment by EmmanuelMess** [comment]
Related #2459
